tests/sentry/deletions/test_group.py

```diff
@@ -22,40 +22,24 @@
 class DeleteGroupTest(TestCase, SnubaTestCase):
     def setUp(self):
         super().setUp()
-        self.event_id = "a" * 32
```

Comment:
These are no-op changes which can be reviewed more easily with the split view.

<img width="217" alt="image" src="https://github.com/user-attachments/assets/1d3d1aa1-cd22-4547-8e51-38dd5265a11d">